pkgs/development/ocaml-modules/owee/default.nix

```
@@ -7,11 +7,11 @@
 buildDunePackage rec {
   minimalOCamlVersion = "4.06";
```

```suggestion
  minimalOCamlVersion = "4.08";
```

https://github.com/let-def/owee/compare/v0.7...v0.8